server/src/internalClusterTest/java/org/opensearch/nodestats/NodeStatsIT.java

```diff
@@ -273,4 +488,22 @@ private void updateExpectedDocStatusCounter(Exception e) {
         expectedDocStatusStats.inc(ExceptionsHelper.status(e));
     }
 
+    private enum MockFields {
```

Comment:
Do we need MockFields ? Can we instead use `NodeIndicesStats.Fields` ?

Reply:
I added additional unknown fields such as `null` and `unknown` to test the working. These are not needed in `NodeIndicesStats.Fields` and hence I introduced  `MockFields`.